spec/memory-safe-d.dd

```diff
@@ -38,7 +38,7 @@ $(H2 $(LNAME2 usage, Usage))
         $(P `@trusted` functions have all the capabilities of `@system` functions but may be called from
         `@safe` functions. For this reason they should be very limited in the scope of their use. Typical uses of
         `@trusted` functions include wrapping system calls that take buffer pointer and length arguments separately so that
-        @safe` functions may call them with arrays.)
+        @safe` functions may call them with arrays. `@trusted` functions must have an `@safe` interface.)
```

Comment:
It might be better to phrase this as:

```
`@trusted` functions must have a safe interface
```

or:

```
`@trusted` functions must have a memory-safe interface
```

to avoid confusion between safe and `@safe` (if that makes sense).  Cf. how things were phrased in the original version of the spec (link in separate comment by me).

Reply:
Would it be useful to provide an example of a "memory-safe interface"?

Reply:
> Would it be useful to provide an example of a "memory-safe interface"?

We already have examples on another page: <https://dlang.org/spec/function.html#safe-interfaces>. Don't repeat all that. Just link there.